src/renderers/dom/shared/__tests__/ReactDOMComponent-test.js

```diff
@@ -331,6 +400,99 @@ describe('ReactDOMComponent', function() {
       expect(container.childNodes[0].getAttribute('myattr')).toBe('myval');
     });
 
+    it('should update known hyphenated attributes for SVG tags', function() {
+      var container = document.createElement('div');
+
+      var beforeUpdate = React.createElement('svg', {}, null);
```

Comment:
Weird that these tests you copied don't use JSX. Mind changing the new ones to use JSX?